{"name": "Try to search -000575", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //a[contains(text(), 'MI-000575 Карнавальная ночь')]\n\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1456, in run\n    match.run(runner.context)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1903, in run\n    self.func(context, *args, **kwargs)\n  File \"step_test.py\", line 61, in step\n    \"//%s[contains(text(), '%s')]\" % (tag, text))\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 855, in find_element\n    'value': value})['value']\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 308, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 194, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Then clear element by id 'search_input'", "status": "passed", "start": 1518158527177, "stop": 1518158527209}, {"name": "Then insert by id 'search_input' '-000575'", "status": "passed", "start": 1518158527209, "stop": 1518158527224}, {"name": "Then page wait '2'", "status": "passed", "start": 1518158527224, "stop": 1518158529232}, {"name": "Then find element 'a' with text 'MI-000575 Карнавальная ночь'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //a[contains(text(), 'MI-000575 Карнавальная ночь')]\n\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1456, in run\n    match.run(runner.context)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1903, in run\n    self.func(context, *args, **kwargs)\n  File \"step_test.py\", line 61, in step\n    \"//%s[contains(text(), '%s')]\" % (tag, text))\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 855, in find_element\n    'value': value})['value']\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 308, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 194, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1518158529232, "stop": 1518158529232}], "attachments": [{"name": "screenshot", "source": "6aba0652-6294-4cec-aad3-68a98f8db6f8-attachment.png", "type": "image/png"}], "start": 1518158527177, "stop": 1518158529470, "uuid": "fb5f5174-d38c-4078-9e85-64421dd9c48e", "historyId": "d26ecb6d81e67a9dd05de89aa1f69435", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Checking qsearch (step)"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}